nano/node/node.cpp

```diff
@@ -305,6 +322,22 @@ void nano::network::republish_block (std::shared_ptr<nano::block> block)
 	}
 }
 
+void nano::network::republish_block (std::shared_ptr<nano::block> block, nano::endpoint const & peer_a)
+{
+	auto hash (block->hash ());
+	nano::publish message (block);
+	std::shared_ptr<std::vector<uint8_t>> bytes (new std::vector<uint8_t>);
```

Comment:
This can be a stack variable

Reply:
Probably doesn't make much sense here because nano::network::republish accept std::shared_ptr<std::vector<uint8_t>> buffer_a

Reply:
Yeh fair enough, I didn't see that